src/server/routes/entity/edition.js

```diff
@@ -256,7 +256,7 @@ function editionToFormState(edition) {
 			null : entityToOption(edition.publisherSet.publishers[0])
 	);
 
-	const publication = entityToOption(edition.publication);
+	const publication = !edition.publicationBbid ? null : entityToOption(edition.publication);
```

Comment:
I'd rather see a fix to `return null` if there is no entity passed to `entityToOption`, that way we protect against other misuses.

There are other `entityToOption` functions in other components where the same fix would be useful, if you search for them.

Reply:
> that way we protect against other misuses

What I mean by that is that if `entityToOption` is called with a null or undefined or empty object somewhere else in the file, now or in future code, we won't run into the same problem of remembering to have to check if it exists.

Doing the same in other components that have `entityToOption` is the same idea: defend against non-existing entity rather than risk throwing an error.